nomad/grapl-core.nomad

```diff
@@ -1354,16 +1354,17 @@ job "grapl-core" {
       }
 
       env {
-        AWS_REGION                          = var.aws_region
-        NOMAD_SERVICE_ADDRESS               = "${attr.unique.network.ip-address}:4646"
-        PLUGIN_REGISTRY_BIND_ADDRESS        = "0.0.0.0:${NOMAD_PORT_plugin-registry-port}"
-        PLUGIN_REGISTRY_DB_HOSTNAME         = var.plugin_registry_db_hostname
-        PLUGIN_REGISTRY_DB_PASSWORD         = var.plugin_registry_db_password
-        PLUGIN_REGISTRY_DB_PORT             = var.plugin_registry_db_port
-        PLUGIN_REGISTRY_DB_USERNAME         = var.plugin_registry_db_username
-        PLUGIN_BOOTSTRAP_CONTAINER_IMAGE    = var.container_images["plugin-bootstrap"]
-        PLUGIN_REGISTRY_KERNEL_ARTIFACT_URL = var.plugin_registry_kernel_artifact_url
-        PLUGIN_REGISTRY_ROOTFS_ARTIFACT_URL = var.plugin_registry_rootfs_artifact_url
+        AWS_REGION                                      = var.aws_region
+        NOMAD_SERVICE_ADDRESS                           = "${attr.unique.network.ip-address}:4646"
+        PLUGIN_REGISTRY_BIND_ADDRESS                    = "0.0.0.0:${NOMAD_PORT_plugin-registry-port}"
+        PLUGIN_REGISTRY_DB_HOSTNAME                     = var.plugin_registry_db_hostname
+        PLUGIN_REGISTRY_DB_PASSWORD                     = var.plugin_registry_db_password
+        PLUGIN_REGISTRY_DB_PORT                         = var.plugin_registry_db_port
+        PLUGIN_REGISTRY_DB_USERNAME                     = var.plugin_registry_db_username
+        PLUGIN_BOOTSTRAP_CONTAINER_IMAGE                = var.container_images["plugin-bootstrap"]
+        PLUGIN_REGISTRY_KERNEL_ARTIFACT_URL             = var.plugin_registry_kernel_artifact_url
+        PLUGIN_REGISTRY_ROOTFS_ARTIFACT_URL             = var.plugin_registry_rootfs_artifact_url
+        PLUGIN_REGISTRY_HAX_DOCKER_PLUGIN_RUNTIME_IMAGE = var.container_images["hax-docker-plugin-runtime"]
```

Comment:
this 'hax' is carrying over a pattern commonly used at my previous company for basically blatantly signaling to everybody who sees a piece of code that "this is going away soon and is not necessarily high quality, by intention"